docs/sqlserver.md

```diff
@@ -0,0 +1,55 @@
+# `sqlserver` Metrics Receiver
+
+The `sqlserver` metrics receiver can fetch stats from a SQL Server via the [Windows Performance Counters](https://docs.microsoft.com/en-us/windows/win32/perfctrs/performance-counters-portal).
```

Comment:
I'll let @qingling128 make the final determination here, but does it make sense to call out the differences between this receiver and `mssql`? Otherwise it's going to be very confusing to customers to have two receivers with seemingly the same functionality. If we do call out the difference, we should add some text to the mssql docs as well.

Reply:
Similar to the [new `iis` metrics](https://github.com/GoogleCloudPlatform/ops-agent/pull/534/files#diff-98d6406bd3b368722a5ecde561e446b313751caddc7c593b5ae130467728cbcfR37), this should be under the existing `mssql` receiver with a `receiver_version: 2` field, rather than as a separate `sqlserver` receiver.

Reply:
This was not quite called out in the scope doc "This will not replace the existing mssql metric functionality and will be able to be run alongside it." But I'll wait for @qingling128's feedback and if you're all in agreement then I'll move forward with this pattern.